docs/source/Tutorial.md

```diff
@@ -947,11 +1108,78 @@ Here is a repetition these lines, to help highlight them more clearly:
     \MyGame\Sample\Monster::AddEquipped($builder, $axe); // Union data
   ~~~
 </div>
+<div class="language-c">
```

Comment:
This is all very long. Please keep this file for the most common usage of the C interface, and move any discussion of alternatives and API options to CUsage.md if needed.